modules/reactor/test/tests/meshgenerators/core_mesh_generator/tests

```diff
@@ -124,4 +124,18 @@
     exodiff = 'core_single_assembly_hex_empty.e'
     recover = false
   []
+  [ptmg_periphery]
+    requirement = 'The system shall generate a 2D hex core mesh with a reactor periphery generated from PeripheryTriangleMeshGenerator (PTMG).'
+    type = 'CSVDiff'
+    input = 'core_periphery_ptmg_vol.i'
+    csvdiff = 'core_periphery_ptmg_in.csv'
+    recover = false
+  []
+  [prmg_periphery]
+    requirement = 'The system shall generate a 2D hex core mesh with a reactor periphery generated from PeripheryRingMeshGenerator (PRMG).'
+    type = 'Exodiff'
+    input = 'core_periphery_prmg.i'
+    exodiff = 'core_periphery_prmg_in.e'
```

Comment:
this file is on the heavier side because of the fields you are outputting.

if you add
cli_args = '--mesh-only'
you ll test the mesh, no extra field